front/src/services/api-http-interceptor.service.js

```diff
@@ -0,0 +1,44 @@
+import { apiSuffix, baseUrl } from '@config';
+
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: `${baseUrl}/${apiSuffix}`,
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json'
+  }
+});
+
+instance.interceptors.request.use(
+  config => {
+    const token = JSON.parse(localStorage.getItem('user'));
+    if (token) {
+      config.headers.common.Authorization = `Bearer ${token.accessToken}`;
+    }
+    return config;
+  },
+  error => {
+    return Promise.reject(error);
+  }
+);
+
+instance.interceptors.response.use(
+  response => {
+    if (response.status === 200 || response.status === 201) {
+      return Promise.resolve(response);
+    } else {
+      return Promise.reject(response);
+    }
+  }, error => {
+    if (error.response.status) {
+      switch (error.response.status) {
+        case 400:
```

Comment:
why do we need this "case" if it acts exactly the same as default return

Reply:
This is an example for the guys to understand what is happening here. But I can remove it

Reply:
It just seemed to me that it would be clearer

Reply:
yes, it seems it could be removed and merged))) I dream to merge it